src/compiler-documenting.md

```diff
@@ -35,28 +35,18 @@ Much like individual tests or building certain components you can build only
 
 ## Document internal rustc items
 
-Compiler documentation is not built by default. There's a flag in
-config.toml for achieving the same.
-But, when enabled, compiler documentation does include internal items.
+Compiler documentation is not built by default. To enable it, modify config.toml:
 
-Next open up config.toml and make sure these two lines are set to true:
-
-```bash
-docs = true
+```toml
+[build]
 compiler-docs = true
 ```
 
-When you want to build the compiler docs as well run this command:
-
-```bash
-./x.py doc
```

Comment:
Do we have this command documented somewhere?

Reply:
it is, beginning of the document